Core/include/Acts/Seeding/SeedFinderOrthogonalConfig.hpp

```diff
@@ -0,0 +1,110 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Units.hpp"
+
+#include <memory>
+
+namespace Acts {
+// forward declaration to avoid cyclic dependence
+template <typename T>
+class SeedFilter;
+
+template <typename SpacePoint>
+struct SeedFinderOrthogonalConfig {
+  std::shared_ptr<Acts::SeedFilter<SpacePoint>> seedFilter;
+
+  // Seed Cuts
+  // lower cutoff for seeds
+  float minPt = 400. * Acts::UnitConstants::MeV;
+  // cot of maximum theta angle
+  // equivalent to 2.7 eta (pseudorapidity)
+  float cotThetaMax = 7.40627;
+  // minimum distance in r between two measurements within one seed
+  float deltaRMin = 5 * Acts::UnitConstants::mm;
+  // maximum distance in r between two measurements within one seed
+  float deltaRMax = 270 * Acts::UnitConstants::mm;
+
+  // impact parameter
+  float impactMax = 20. * Acts::UnitConstants::mm;
+
+  // how many sigmas of scattering angle should be considered?
+  float sigmaScattering = 5;
+  // Upper pt limit for scattering calculation
+  float maxPtScattering = 10 * Acts::UnitConstants::GeV;
+
+  // for how many seeds can one SpacePoint be the middle SpacePoint?
+  unsigned int maxSeedsPerSpM = 5;
+
+  // Geometry Settings
+  // Detector ROI
+  // limiting location of collision region in z
+  float collisionRegionMin = -150 * Acts::UnitConstants::mm;
+  float collisionRegionMax = +150 * Acts::UnitConstants::mm;
+  float phiMin = -M_PI;
+  float phiMax = M_PI;
+  // limiting location of measurements
+  float zMin = -2800 * Acts::UnitConstants::mm;
+  float zMax = 2800 * Acts::UnitConstants::mm;
+  float rMax = 600 * Acts::UnitConstants::mm;
+  // WARNING: if rMin is smaller than impactMax, the bin size will be 2*pi,
+  // which will make seeding very slow!
+  float rMin = 33 * Acts::UnitConstants::mm;
+
+  float rMinMiddle = 60.f * Acts::UnitConstants::mm;
+  float rMaxMiddle = 120.f * Acts::UnitConstants::mm;
+
+  float deltaPhiMax = 0.085;
+
+  float bFieldInZ = 2.08 * Acts::UnitConstants::T;
+  // location of beam in x,y plane.
+  // used as offset for Space Points
+  Acts::Vector2 beamPos{0 * Acts::UnitConstants::mm,
```

Comment:
One more comment we should follow up, the beam position actually should be contextual to the event, but again - something for later.

Reply:
That's actually a good point. I believe the *conventional* seed finder currently also has this in the config. This should be passed per invocation at some point.